tutorials/monitoring-setup/index.md

```diff
@@ -123,11 +135,13 @@ NOTE: Creating the Monitoring instance will also create the monitoring queue if
 
 ### Install ServicePulse
 
+NOTE: Make sure that ServiceControl and ServiceControl.Monitoring instances are added first.
+
 [ServicePulse](/servicepulse/) is a web application for production monitoring and recoverability. It connects to a Monitoring instance to display monitoring data and to a ServiceControl instance to display recoverability data.
 
 Download and run the latest [ServicePulse installer](https://github.com/Particular/ServicePulse/releases/download/1.10.1/Particular.ServicePulse-1.10.1.exe). 
 
-NOTE: If you already have ServicePulse running on your machine you will need to uninstall it and download the latest version in order to be able to configure ServicePulse to connect to a ServiceControl Monitoring instance.
+NOTE: In order to be able to configure ServicePulse to connect to a ServiceControl Monitoring instance you must download the latest version, then uninstall your current version and install the download version.
```

Comment:
If they are using the platform installer, it may not be possible to do these steps in the order listed. Should we suggest uninstalling the current version first?